Utils/github_workflow_scripts/send_slack_message.py

```diff
@@ -0,0 +1,219 @@
+
+        Args:
+            key (str): pack related key (example pack version)
+            value (str): pack related value (example 1.0.0)
+
+        Returns:
+            (dict): section entry for the slack block-kit
+    """
+    return ({
+        "type": "section",
+        "text": create_slack_markdown(f'```{key}: {value}\n```')
+    })
+
+
+def create_individual_pack_segment(metadata_obj: dict) -> List[dict]:
+    """Create the pack information segment of the message
+
+        Args:
+            metadata_obj (dict): metadata information dictionary
+
+        Returns:
+            (list): List of slack blocks representing the pack information
+    """
+    pack_name = metadata_obj.get('name')
+    version = metadata_obj.get('currentVersion')
+    support = metadata_obj.get('support')
+
+    pack_details = [
+        create_slack_section('Pack Name', pack_name),
+        create_slack_section('Support Type', support),
+        create_slack_section('Version', version),
+        {
+            "type": "divider"
+        }
+    ]
+    return pack_details
+
+
+def create_packs_segment(metadata_files: PaginatedList) -> List[dict]:
+    """Aggregate the pack information segments of the message
+
+        Args:
+            metadata_files (PaginatedList): List of File objects representing metadata files
+
+        Returns:
+            (list): List of slack blocks representing all packs information
+    """
+    all_packs = []
+    for file in metadata_files:
+        metadata_obj = get_metadata_file(file)
+        pack_segment = create_individual_pack_segment(metadata_obj)
+        all_packs += pack_segment
+    return all_packs
+
+
+def create_pull_request_segment(pr: PullRequest) -> List[dict]:
+    """Create the pull request information segment of the message
+
+        Args:
+            pr (PullRequest): object that represents the pull request.
+
+        Returns:
+            (list): List containing a slack block-kit section entry which represents the PR info
+    """
+    assignees = ','.join([assignee.login for assignee in pr.assignees])
+    contributor = pr.user.login
+    number_of_changed_changed_files = pr.changed_files
+    labels = ','.join([label.name for label in pr.labels])
+    pr_info_segment = create_slack_fields([
+        ('Assignees', assignees),
+        ('Contributor', contributor),
+        ('Changed Files', number_of_changed_changed_files),
+        ('Labels', labels),
+    ])
+    return [pr_info_segment]
+
+
+def create_pr_title(pr: PullRequest) -> List[dict]:
+    """Create the message title
+
+        Args:
+            pr (PullRequest): object that represents the pull request.
+
+        Returns:
+            (list): List containing a dictionary which represents the message title
+    """
+    header = [{
+        "type": "header",
+        "text": {
+            "type": "plain_text",
+            "text": f"{pr.title}",
+            "emoji": True
+        }
```

Comment:
```suggestion
        "text": create_slack_markdown(f"<{pr.html_url}|{pr.title}>")
```
Let's just do this and either remove line 183 or make it the same value as what we have here.

Reply:
1. Using `create_slack_markdown` to create the text fo the header would not work (it needs `"type": "plain_text"` and not ` "type": "mrkdwn"` as being create in `create_slack_markdown`)
2. If you will look at the image I attached you will see that the header is different from the text created in line 183 (We have both because a header cannot contain hyperlink). Maybe it is worth removing the header but I think it adds good structure to the message.

Reply:
True. Fixed.